modules/get_primary_metadata_region/variables.tf

```diff
@@ -0,0 +1,15 @@
+##############################################################################
+# Input Variables
+##############################################################################
+
+variable "region" {
+  description = "The IBM Cloud Metrics Routing region."
```

Comment:
A region input variable doesn't really make sense for this module? The goal of the module is to find out if the account has a primary metadata region already set. 
So var.region makes no sense here. I can see you are using it to construct the api. But does it matter what region endpoint is even used here? Will they all return the same result? Or even better, is there a global api?